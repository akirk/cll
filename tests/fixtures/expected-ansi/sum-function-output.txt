Absolutely! A common example is proving the formula for the sum of the first n natural numbers:

\[
S(n) = 1 + 2 + 3 + \ldots + n = \frac{n(n + 1)}{2}
\]

Let's go through this step by step.

[1mStep 1: Base Case[m

We need to verify that this formula holds when n = 1. Can you calculate S(1) using the formula and the sum of the natural numbers? What do you find?