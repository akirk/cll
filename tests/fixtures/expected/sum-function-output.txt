Absolutely! A common example is proving the formula for the sum of the first n natural numbers:

\[
S(n) = 1 + 2 + 3 + \ldots + n = \frac{n(n + 1)}{2}
\]

Let's go through this step by step.

Step 1: Base Case

We need to verify that this formula holds when n = 1. Can you calculate S(1) using the formula and the sum of the natural numbers? What do you find?